cnxpublishing/views/__init__.py

```diff
@@ -33,6 +33,17 @@ def declare_api_routes(config):
     add_route('api-keys', '/api-keys')
     add_route('api-key', '/api-keys/{id}')
 
+    # Print Style routes
+    add_route('print-style-history',
+              '/status/print-style-history',)
+    add_route('print-style-history-name',
+              '/status/print-style-history/{name}')
+    # add_route('print-style-history-name_POST',
+    #           '/status/print-style-history/{name}', request_method='POST')
+    add_route('print-style-history-version',
+              '/status/print-style-history/{name}/{version}')
+    add_route('resource', '/resources/{hash}{ignore:(/.*)?}')  # noqa cnxarchive.views:get_resource
```

Comment:
I'm not sure where the `/status/print-style-history` bit of this came from. Let's change these to the original spec: `/print-styles/...`.